runtime-sdk/modules/evm/src/lib.rs

```diff
@@ -76,7 +76,7 @@ pub trait Config: 'static {
     /// If any of the precompile addresses returned is the same as for one of
     /// the builtin precompiles, then the returned implementation will
     /// overwrite the builtin implementation.
-    fn additional_precompiles() -> Option<precompile::PrecompileSetType> {
+    fn additional_precompiles() -> Option<()> {
```

Comment:
Hm, why this change? I assume it is because the new precompile set type is now pretty hard to provide externally, but this seems like it makes the function useless as it is now impossible to specify additional precompiles.

I believe this is not currently used anywhere so I would be fine with just removing the function for now if you think this can no longer be implemented.

Reply:
I didn't see that this was part of a trait. I added the option back.